pkg/fileutils/fileutils_test.go

```diff
@@ -276,3 +276,16 @@ func (s *FileUtilTestSuite) TestLoadYaml(c *check.C) {
 
 	}
 }
+
+func (s *FileUtilTestSuite) TestIsRegularFile(c *check.C) {
+	pathStr := filepath.Join(s.tmpDir, "TestIsRegularFile")
+	c.Assert(IsRegularFile(pathStr), check.Equals, false)
+
+	os.Create(pathStr)
+	c.Assert(IsRegularFile(pathStr), check.Equals, true)
+	os.Remove(pathStr)
+
+	os.OpenFile(pathStr, 0, 0666)
+	c.Assert(IsRegularFile(pathStr), check.Equals, false)
```

Comment:
could you add a comment here why it equals to false? It makes more readable.

Reply:
updated, PTAL.